app/models/fee_structure.rb

```diff
@@ -75,8 +75,8 @@ def calculate
       (BigDecimal(amount) * fee_surcharge).ceil + fee_structure.flat_fee + flat_fee
     end
 
-    def self.calculate(**args)
-      FeeCalculation.new(**args).calculate
+    def self.calculate(args={})
```

Comment:
I can never remember how the double splat and hash stuff works... I always just need to try it and see the outcome...

Reply:
Totally agree. I feel like I am getting better at intuitively knowing but I still mess it up.